src/dense/interpolants.jl

```diff
@@ -366,11 +366,10 @@ simplifying assumption
 
 Ch. Tsitouras
 """
-@def tsit5unpack begin if typeof(cache) <: OrdinaryDiffEqMutableCache
-    @unpack r11, r12, r13, r14, r22, r23, r24, r32, r33, r34, r42, r43, r44, r52, r53, r54, r62, r63, r64, r72, r73, r74 = cache.tab
-else
-    @unpack r11, r12, r13, r14, r22, r23, r24, r32, r33, r34, r42, r43, r44, r52, r53, r54, r62, r63, r64, r72, r73, r74 = cache
-end end
+@def tsit5unpack begin
+    var"#T#" = recursive_unitless_bottom_eltype(y₁)
```

Comment:
I think the types are all missing the `constvalue` part which is crucial for dropping units, ADness, and complex-ness